res/configuration.toml

```diff
@@ -61,9 +61,18 @@ Type = "consul"
   Port = 59881
 
   [Clients.core-command]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59882
+  #  Protocol = "http"
+  #  Host = "localhost"
+  #  Port = 59882
+  # Choose to use new messaging based communication for Command Client.
+  # Only valid if MessageBus Trigger is used
+  # If prefer REST, remove below and uncomment above
```

Comment:
Let remove this. I left it in the template app for awareness for custom app service developers, but here we are choosing the use MessageBus.

Reply:
I have removed the comments